code/platformio_ota.ini

```diff
@@ -246,6 +246,9 @@ extends = env:maxcio-wus002s
 [env:maxcio-wde004-ota]
 extends = env:maxcio-wde004
 
+[env:maxcio-wuk007s-ota]
+extends = env:maxcio-wuk007s
+
```

Comment:
btw note the comment at the top
but, it is still here until I fix the documentation